lib/graphql/server/queries/dmpDataQueries.ts

```diff
@@ -127,6 +129,18 @@ export async function getData() {
     };
     return formData;
   } catch (error: any) {
-    console.log(`Something went wrong: ${error.message}`)
+    const err = JSON.parse(JSON.stringify(error));
```

Comment:
Here I am showing that we could extract error codes in the graphql request methods when there is an error, and map to a user-friendly message, log the error, and then throw the error. The error needs to be thrown here in order for it to be caught in the client-side or server-side component calling these methods.